apps/src/code-studio/components/HiddenUploader.jsx

```diff
@@ -26,6 +28,18 @@ export default class HiddenUploader extends React.Component {
       add: function (e, data) {
         // onUploadStart method must call data.submit()
         props.onUploadStart(data);
+        const audioFileName = data.files[0].name.includes('mp3') ? data.files[0].name : null;
+        if (audioFileName) {
+          firehoseClient.putRecord(
+            {
+              study: 'sound-dialog',
+              study_group: experiments.isEnabled(experiments.AUDIO_LIBRARY_DEFAULT) ? 'library-tab' : 'files-tab',
+              event: 'upload-file',
+              data_json: audioFileName,
+            },
+            {includeUserId: true}
```

Comment:
Should this differentiate between user-recorded mp3s and other uploaded mp3s?

Reply:
Chatted with Poorva around this and we don't need to differentiate between recorded/uploaded since the intent of this A/B test is to determine whether we should default the sound dialog to the sound library or the sound files tab. The logging will also let us know if there are sounds that students are adding manually because they are missing from our library. Accordingly, whether a chosen uploaded files is recorded or uploaded, shouldn't matter.